pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -512,9 +513,18 @@ func (r *ReconcileClusterDeployment) reconcile(request reconcile.Request, cd *hi
 				cdLog.Errorf("error creating job: %v", err)
 				return reconcile.Result{}, err
 			}
-			kickstartDuration := time.Since(cd.CreationTimestamp.Time)
-			cdLog.WithField("elapsed", kickstartDuration.Seconds()).Info("calculated time to install job seconds")
-			metricInstallDelaySeconds.Observe(float64(kickstartDuration.Seconds()))
+			if value := cd.Annotations[firstTimeInstallAnnotation]; value == "" {
```

Comment:
a slightly better check would be:
`if _, ok := cd.Annotations[firstTimeInstallAnnotation]; !ok {`